cruise.umple/test/cruise/umple/compiler/024_multiAssocToSameClassWithOneRoleName.ump

```diff
@@ -0,0 +1,7 @@
+class Person{
+}
+
+class Course{
+  * courseProf -- 1 Person prof;
+  * -- 1..* Person students;
+}
```

Comment:
consider testing more example in the both sides.
For example, 
```
class A{
  0..1  a1-- * B;
  0..1  a2-- * B;
}
class B{}
```
will give the following message in which we have a asociation between `A` and  `A`.
```
Error 19 on line 3 of file "test.ump":
There are multiple associations between class 'A' and class 'A'. Role names need to be added at 'A' side to distinguish the different association ends in that class

```

Reply:
Any improvement on this? The association should be between `A` an `B`.